src/Core/Models/SqlTypeConstants.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Service.Models
```

Comment:
Let's start using [file-scoped namespaces](https://learn.microsoft.com/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces) to reduce the bloat in files.

Easy refactor in VS, just put a `;` on the end of line 4 and it'll fix the rest up for you.